GeosCore/vdiff_mod.F90

```diff
@@ -1898,6 +1903,105 @@ SUBROUTINE VDIFFDR( Input_Opt,  State_Chm, State_Diag, &
     ! Convert kg/kg -> g/kg
     p_shp    = p_shp * 1.0e+3_fp
 
+#ifdef LUO_WETDEP
+!$OMP PARALLEL DO DEFAULT( SHARED )      &
+!$OMP PRIVATE( I, J, L, F_CLD ) &
+!$OMP PRIVATE( YCLDICE, FICE, YB, volx34pi_cd, rrate, SQM, STK ) &
+!$OMP PRIVATE( DFKG, uptkrate )
+    DO L = 1, State_Grid%NZ
+    DO J = 1, State_Grid%NY
+    DO I = 1, State_Grid%NX
+
+      IF(L>State_Met%PBL_TOP_L(I,J))THEN
+      State_Met%WUP(I,J,L)=(0.5D0*(kvh(I,J,L)+kvh(I,J,L+1))/30.D0/&
+                            0.516D0)*SQRT(2.D0/3.D0)
+      ELSE
+      State_Met%WUP(I,J,L)=(0.5D0*(kvh(I,J,L)+kvh(I,J,L+1))/&
+           MAX(30.D0,(1.D0/(1.D0/(0.4D0*SUM(State_Met%BXHEIGHT(I,J,1:L)))+&
+                            1.D0/300.D0)))/&
+                            0.516D0)*SQRT(2.D0/3.D0)
```

Comment:
Recommendation: replace `SQRT(2.D0/3.D0)` into a defined parameter
```F90
REAL(fp), PARAMETER :: ROOT_TWO_THIRDS = SQRT(2.D0/3.D0)
```
so that the computation is only done once and stored for successive uses.